crates/chia-consensus/src/test_generators.rs

```diff
@@ -303,32 +305,49 @@ fn run_generator(#[case] name: &str) {
         ]
         .contains(&name)
         {
-            // test that we allow quoted generators with the flag
-            flags |= SIMPLE_GENERATOR;
```

Comment:
It should be worth keeping this flag setting so we can check that all the simple generators pass with it set and then you won't need to also specifically check the node check in the test either as it will be checked as part of `run_block_generator2()`